test/integration/policy_kyverno_generators_test.go

```diff
@@ -245,67 +268,88 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the kyverno generator policie
 
 		// delete the policies
 		for _, url := range policyNameMap {
-			utils.KubectlWithOutput(
+			_, err := utils.KubectlWithOutput(
 				"delete", "-f", url, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
 			)
+			if !k8serrors.IsNotFound(err) {
+				var exitError *exec.ExitError
+				ok := errors.As(err, &exitError)
+				if ok {
+					Expect(exitError.Stderr).To(BeNil())
+				} else {
+					Expect(err).To(BeNil())
+				}
+			}
 		}
 
 		// remove the kyverno install policy
-		utils.KubectlWithOutput(
+		_, err := utils.KubectlWithOutput(
 			"delete", "-f", kyvernoInstallURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
 		)
+		if !k8serrors.IsNotFound(err) {
+			var exitError *exec.ExitError
+			ok := errors.As(err, &exitError)
+			if ok {
+				Expect(exitError.Stderr).To(BeNil())
+			} else {
+				Expect(err).To(BeNil())
+			}
+		}
 
 		// delete the subscription
-		utils.KubectlWithOutput(
+		_, err = utils.KubectlWithOutput(
 			"delete", "subscription.apps.open-cluster-management.io", "-n", kyvernoNamespace, "--all",
 			"--kubeconfig="+kubeconfigManaged,
 		)
+		if !k8serrors.IsNotFound(err) {
+			var exitError *exec.ExitError
+			ok := errors.As(err, &exitError)
+			if ok {
+				Expect(exitError.Stderr).To(BeNil())
+			} else {
+				Expect(err).To(BeNil())
+			}
+		}
 
 		// delete the namespace created to test the generators
-		utils.KubectlWithOutput(
+		_, _ = utils.KubectlWithOutput(
```

Comment:
The error should be checked after the `--ignore-not-found` flag is added. This comment applies to the rest of them.